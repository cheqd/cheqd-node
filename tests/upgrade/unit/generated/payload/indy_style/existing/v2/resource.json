{
    "metadata": {
        "collectionId": "xcWigjiDhDTaBGRX",
        "id": "7f2589aa-1279-4bd7-b704-f868772b460a",
        "name": "Test Resource",
        "resourceType": "test-resource-type",
        "mediaType": "application/json",
        "created": "2022-11-01T10:54:01Z",
        "checksum": "T0ebujH3GFnZlDZRU4yN8hGYBOzrf1ig3iQXif8hSvQ=",
        "previousVersionId": null,
        "nextVersionId": null
    },
    "resource": {
        "data": "eyAibWVzc2FnZSI6ICJoZWxsbyB3b3JsZCJ9"
    }
}