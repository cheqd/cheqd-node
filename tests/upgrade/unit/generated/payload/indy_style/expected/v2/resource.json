{
    "metadata": {
        "collection_id": "7Hty8uGafcEsU7EpJEAPUC",
        "id": "7f2589aa-1279-4bd7-b704-f868772b460a",
        "name": "Test Resource",
        "resource_type": "test-resource-type",
        "media_type": "application/json",
        "created": "2022-11-01T10:54:01Z",
        "checksum": "T0ebujH3GFnZlDZRU4yN8hGYBOzrf1ig3iQXif8hSvQ=",
        "previous_version_id": null,
        "next_version_id": null
    },
    "resource": {
        "data": "eyAibWVzc2FnZSI6ICJoZWxsbyB3b3JsZCJ9"
    }
}